autovalet_lane_detection/src/autovalet_lane_detection/LaneDetector.py

```diff
@@ -211,7 +212,10 @@ def findNormalVectorInCloud(self, center_line_cloud):
         # vh[-1] corresponds to the normal vector of the center line, but not unique since this is a 3D world.
         # taking (z, -x) s.t. taking inner product with (x, z) = 0
         _, _, vh = np.linalg.svd(center_line_cloud - center_mean)
-        orth_vec = np.sign(vh[0][2]) * np.array([vh[0][2], -vh[0][0]])
+        # principle_dir = np.sign(vh[0][2])*vh[0]
+        #orth_vec = np.sign(vh[0][2] + np.sin(np.pi/4)) * np.array([vh[0][2], -vh[0][0]])
```

Comment:
get rid of commented lines we're not using

Reply:
you cool